frontend/integration-tests/tests/kubevirt/vm.nic.binding.scenario.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-undef */
+import { $, browser } from 'protractor';
+
+import { appHost, testName } from '../../protractor.conf';
+import { isLoaded } from '../../views/crud.view';
+import { testNad, basicVmConfig, networkInterface, networkBindingMethod, networkWizardTabCol, networkTabCol } from './mocks';
+import * as vmView from '../../views/kubevirt/virtualMachine.view';
+import { removeLeakedResources, execCommandFromCli, assertUnorderListEqual, fillInput, selectDropdownOption, getDropdownOptions } from './utils';
+import Wizard from './models/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import * as wizardView from '../../views/kubevirt/wizard.view';
+
+describe('Test VM network interface binding method', () => {
+  const leakedResources = new Set<string>();
+  const wizard = new Wizard();
+  const vmName = `vm-${testName}`;
+  const vm = new VirtualMachine(vmName, testName);
+  const podNetworkList = [networkBindingMethod.pod, networkBindingMethod.default, networkBindingMethod.sriov];
+  const networkList = [networkBindingMethod.default, networkBindingMethod.sriov];
+
+  beforeAll(async() => {
+    execCommandFromCli(`echo '${JSON.stringify(testNad)}' | kubectl create -f -`);
+  });
+
+  afterAll(async() => {
+    execCommandFromCli(`echo '${JSON.stringify(testNad)}' | kubectl delete -f -`);
+    execCommandFromCli(`kubectl delete -n ${testName} vm ${vmName}`);
+    leakedResources.delete(JSON.stringify({name: vmName, namespace: testName, kind: 'vm'}));
+    removeLeakedResources(leakedResources);
+  });
+
+  it('Check nic binding method on wizard', async() => {
+
+    const provisionSource = {method: 'Container', source: basicVmConfig.sourceContainer};
+
+    await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+    await isLoaded();
+    await wizard.openWizard();
+
+    // Basic Settings for VM
+    await wizard.fillName(vmName);
+    await wizard.fillDescription(testName);
+    await wizard.selectProvisionSource(provisionSource);
+    await wizard.selectOperatingSystem(basicVmConfig.operatingSystem);
+    await wizard.selectFlavor(basicVmConfig.flavor);
+    await wizard.selectWorkloadProfile(basicVmConfig.workloadProfile);
+    await wizard.next();
+
+    // Networking
+    // Pod networking
+    let rowsCount = await wizardView.tableRowsCount();
+    let bindingID = wizardView.dropdownOptionID('binding', rowsCount);
+    const podNetworkBinding = wizardView.itemInTable(0, networkWizardTabCol.binding - 1);
+    // Pod networking default binding is masquerade
+    expect(podNetworkBinding.getText()).toEqual(networkBindingMethod.pod);
+    await podNetworkBinding.click();
```

Comment:
maybe add this comment ? `// open a row`

Reply:
added

Reply:
I meant to add it so it explains why we click on podNetworkBinding twice. Not sure if it is apparent from current comment

Reply:
1st click enable the field, 2nd click get the dropdown list. Update the comment.

Reply:
1st click activates the row

Reply:
done